Runtime/Scripts/Component/Output/OscSender.cs

```diff
@@ -13,10 +13,29 @@ public class OscSender : MonoBehaviour
         [SerializeField] int m_Port = 7000;
 
         /// <summary>The IP address to send to</summary>
-        public string IpAddress => m_IpAddress;
-        
+        public string IpAddress
+        {
+            get { return m_IpAddress; }
+            set {
+                    string[] ipString = value.Split('.');
```

Comment:
i think we can rely on the built-in IP validation, which probably is more reliable than us writing one
https://docs.microsoft.com/en-us/dotnet/api/system.net.ipaddress.tryparse 
```csharp
bool valid = IPAddress.TryParse(value, out var ip);  
```

Reply:
oh! I was wondering how to use that initially. Forgot to add `System.Net` cool.